gtsfm/data_association/data_assoc.py

```diff
@@ -55,6 +59,8 @@ def run(
         cameras: Dict[int, PinholeCameraCal3Bundler],
         corr_idxs_dict: Dict[Tuple[int, int], np.ndarray],
         keypoints_list: List[Keypoints],
+        images: Optional[List[Image]] = None,
+        viz_patch_sz: int = 100,
```

Comment:
I think  visualization should be separate from the validation function.

Reply:
Thanks for taking a look, Ayush.

What did you mean by "validation function"?

Reply:
This function is being used to check when we have a track with minimum acceptable length. I would prefer we write a seperate function for visualization and not merge core logic and visualization in the same function.

Reply:
makes sense, good point